src/status_im/contexts/communities/events.cljs

```diff
@@ -186,3 +187,13 @@
                      :params     []
                      :on-success #(rf/dispatch [:communities/fetched-collapsed-categories-success %])
                      :on-error   #(log/error "failed to fetch collapsed community categories" %)}]}))
+
+
+(rf/reg-event-fx :communities/share-community-channel-url-with-data 
+    (fn [_ [chat-id]]
+      (let [community-id (subs chat-id 0 constants/community-id-length)
+            channel-id (subs chat-id constants/community-id-length)] 
+   {:json-rpc/call [{:method     "wakuext_shareCommunityChannelURLWithData"
+                     :params      [{:CommunityID community-id :ChannelID channel-id}] 
+                     :on-success  #(share/open {:url %}) 
```

Comment:
The default error handling in `share/open` shouldn't be a no-op (line 7 below), we should at least log errors, but given this is not your PR's responsibility, I suggest we at least call `share/open` with an `on-error` callback that logs the error. Swallowing rejections is never a good idea.

https://github.com/status-im/status-mobile/blob/bec51b7d0e3767848e693d433b92083f396f10a6/src/react_native/share.cljs#L7-L12